sklearn/model_selection/_validation.py

```diff
@@ -449,6 +450,13 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
     return_parameters : bool, default=False
         Return parameters that has been used for the estimator.
 
+    split_progress : list or tuple, optional, default: None
+        A list or tuple of format (<current_split_id>, <total_num_of_splits>)
+
+    param_progress : list or tuple, optional, default: None
```

Comment:
Why Param and not candidate?

Reply:
I agree `param` and `candidate` are mixed right now: following also @NicolasHug [comment](https://github.com/scikit-learn/scikit-learn/pull/16935#discussion_r410887505) I will change all `param` into `candidate` then.